docs/publishing/sections/5_analytics_portfolio_site.md

```diff
@@ -206,3 +209,9 @@ build_my_reports:
     git add portfolio/my_report/district_*/ portfolio/my_report/*.yml portfolio/my_report/*.md
     git add portfolio/sites/my_report.yml
 ```
+
+### Delete Portfolio/ Refresh Index Page
+
+When creating a new portfolio and there’s an old version with existing files or content on your portfolio site or in your local environment, it’s important to clean up the old files before adding new content.
```

Comment:
Maybe this should be "When redeploying your portfolio with new content and there’s an old version with existing files or content on your portfolio site or in your local environment, it’s important to clean up the old files before adding new content."